### DISCUSSION TITLE
Can Specific Pages be Refreshed When Updating Dynamic Route Pages with On-Demand Revalidation?

### DISCUSSION THREAD

**Discussion by shape1541** [vercel/next.js#62454] [App Router]
When updating dynamically routed pages implemented using SSG and getStaticPaths with On-Demand Revalidation, is it possible to refresh specific pages only? Or do all pages need to be refreshed at once?

**Comment by markoleavy** [comment]
You can just use `revalidatePath("full/path/name")` in a route handler, as explained [here](https://nextjs.org/docs/app/api-reference/functions/revalidatePath)  .
In app folder you must use `generateStaticParams` instead of `getStaticPath`.
